### PR TITLE
Create GetDaemon function (currently supports only systemd)

### LINKED ISSUE #6: getDaemonInfo() method
Create getDaemonInfo(string daemonName) method

### PR DESCRIPTION
**Pull request by cooligus**
Closes #6

### CODE REVIEW

**Comment by BaderBC** [comment]
Your code doesn't pass github CI test.
Have you thought about how to fix it?

File: src/api/daemon_info.go
Comment (BaderBC) [inline src/api/daemon_info.go:14]: IsActive as well as isEnabled should be boolean type

File: src/api/daemon_info.go
Comment (BaderBC) [inline src/api/daemon_info.go:48]: there should be unknown not "none known"

File: src/api/daemon_info.go
Comment (BaderBC) [inline src/api/daemon_info.go:21]:
```suggestion
	return strings.TrimSpace(string(stdout)), err
```

File: src/api/daemon_info_test.go
Comment (BaderBC) [inline src/api/daemon_info_test.go:14]: Check whether deamonInfo attributes are not empty

**Review by BaderBC — CHANGES_REQUESTED** [review]

File: src/api/daemon_info.go
Comment (cooligus) [reply src/api/daemon_info.go:14]: systemctl is-active returns 5 possible options and is-enabled 3

File: src/api/daemon_info.go
Comment (cooligus) [reply src/api/daemon_info.go:48]: ok, I will change it to no known

File: src/api/daemon_info_test.go
Comment (cooligus) [reply src/api/daemon_info_test.go:14]: ok

**Review by sewe2000 — COMMENTED** [review]
Please use this enum-like thing for init systems. Everything else seems fine.

**Comment by cooligus** [comment]
Closes #6

File: src/api/daemon.go
Comment (BaderBC) [inline src/api/daemon.go:91]: Instead of manually providing our init system (initSystem) I would consider in the future making auto checking is the background which init system is our OS running.
Remember also there is now way currently for rule author to check which init system is running.
For now we can just support only systemd instead of checking anything.

**Review by BaderBC — CHANGES_REQUESTED** [review]
I gave you some suggestions, mostly logistical, not code specific.
P.S. please, make more meaningful commits, its not really funny what are you doing now

File: src/api/daemon.go
Comment (maxidragon) [inline src/api/daemon.go:40]: If you aren't going to do that in this PR please create issue for it

**Review by maxidragon — APPROVED** [review]
Only one thing, LGTM (but please wait for someone else approval before merging)

File: src/api/daemon.go
Comment (BaderBC) [inline src/api/daemon.go:36]:
```suggestion
	}, nil
```

File: src/api/daemon_test.go
Comment (BaderBC) [inline src/api/daemon_test.go:6]: check at least if Daemon.Name is defined

File: src/api/daemon.go
Comment (BaderBC) [inline src/api/daemon.go:12]: export all these fields

**Review by BaderBC — CHANGES_REQUESTED** [review]
Change mentioned, rest LGTM

**Review by BaderBC — APPROVED** [review]
LGTM!

**Pull request merged**
